client/my-sites/themes/theme-options.js

```diff
@@ -66,10 +91,10 @@ const activate = {
 	action: activateAction,
 	hideForTheme: ( state, themeId, siteId ) => (
 		! getCurrentUser( state ) ||
-		( isJetpackSite( state, siteId ) && isJetpackSiteMultiSite( state, siteId ) ) ||
+		isJetpackSiteMultiSite( state, siteId ) ||
```

Comment:
I think we should keep the `isJetpackSite` criterion here, too. Mainly for consistency.